binder/environment.yml

```diff
@@ -21,7 +21,7 @@ dependencies:
 - fzf >=0.42.0
 - importlib-metadata >=4.6.0
 - intervaltree >=3.0.2
-- ipython >=8.13.0,<9.0.0,!=8.17.1
+- ipython >=8.13.0,<10.0.0,!=8.17.1
```

Comment:
Leaving here a not that these constraints depend on bug https://github.com/ipython/ipython/issues/14937. If it affects us then we'd need to skip 9.1 to 9.4 and wait until 9.5 is out.